pocket-dark.user.css

```diff
@@ -259,28 +260,126 @@
     background: var(--mid-gray);
     box-shadow: -10px 0 5px -2px var(--mid-gray);
   }
-  body, html, .story-content h1, .story-content h2, .story-content p,
-  .story-content q, .syndication-article-header,
+  #page .pkt-nav {
+    background-color: var(--black);
+  }
+  .portal-nav, .portal-header {
+    background-color: var(--mid-gray);
+  }
+  .story-footer-container, .publisher-cta .cta-button:hover {
+    background-color: var(--mid-gray);
+  }
+  .overflow-nav {
+    background-color: var(--black);
+  }
+  .story-content h1, .story-content h2, .syndication-article-header {
+    color: var(--light-gray-two);
+  }
+  body, .story-content p, .story-content q,
   .outgoing-links .outgoing-links-article-title {
-    color: #919191;
+    color: var(--icon-gray);
   }
   .article-join-us .message, .syndicated-article-byline, .publisher-cta,
   .publisher-cta .cta-button {
-    color: #505050;
+    color: var(--dark-gray);
+  }
+  .article-join-us a:hover, .story-content a:hover,
+  .more-from-pocket .related-article .details .title:hover {
+    color: var(--sky-blue);
   }
-  .story-content a, .story-content a:visited, .article-join-us a,
+  .more-from-pocket .related-article .details .title, .story-content a,
+  .story-content a:visited, .article-join-us a,
   .syndicated-article-byline .publisher-name a,
   .outgoing-links .outgoing-links-article-title {
-    color: #1eabf9;
+    color: var(--med-blue);
     text-shadow: none;
   }
-  .portal-nav, .portal-header {
-    background-color: transparent;
+  .footer .signoff .publisher-logo,
+  .more-from-pocket .related-article .details .publisher-logo {
+    filter: invert(90%);
   }
-  .story-footer-container, .publisher-cta .cta-button:hover {
+  .best_of_list, .topic-heading, .trending_list {
+    background-color: var(--black);
+  }
+  .portal_list .item {
     background-color: var(--mid-gray);
   }
-  .overflow-nav {
-    background-color: #111;
+  .item_content p {
+    color: var(--icon-gray);
+  }
+  .title_sponsor a, .title a {
+    color: var(--med-blue);
+  }
+  .gsf_device_reminder {
+    background-color: var(--mid-gray);
+    border: 1px solid var(--mid-gray-two);
+  }
+  @media only screen and (min-width: 26em) {
+    .legacy-container {
+      background-color: var(--black);
+    }
+    .legacy-content {
+      background-color: var(--mid-gray);
+    }
+  }
+  .legacy-sidebar h3, .legacy-sidebar h5, .legacy-sidebar h5>a {
+    color: #478f8f;
```

Comment:
Ide just add this to variables in :root =)

Reply:
Will do

Reply:
We need to review the root var shades of the colors, they were just ripped off the marketing stylesheet from pocket.